### PR TITLE
Make the event variants doc more skimmable and include absence and triggers overrides

### PR DESCRIPTION
**Pull request by aleks-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…ng-plan/event-variants**](https://docs-erzc9dnv4.avo-dev.app/data-design/avo-tracking-plan/event-variants) |
|:-:|
|<a href="https://docs-erzc9dnv4.avo-dev.app/data-design/avo-tracking-plan/event-variants">Screenshot of /data-design/avo-tracking-plan/event-variants</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/3Ftkobb-l8h7Th6Mcf3UsUZkdRNsTwS8ZMOMZAtRE5NvsfFbnBvoyioQSTkmIAbUzh-FKpeJsS7gXGIn6WKhBQ.png">(view full size)</a> |







Commit cb0d00e741ec38c3113113e05c5c522c3ce90003 (https://docs-erzc9dnv4.avo-dev.app).

**Comment by Svalavo** [comment]
Looks good to me! Only issue I found was a missing space in the Introduction. Between the period and start of "A few scenarios to mention:"
Screenshot 2024-06-10 at 11 35 11

File: pages/data-design/avo-tracking-plan/event-variants.mdx [hunk 1]
```diff
@@ -3,13 +3,29 @@ import Image from 'next/image'
...
 
-Event variants streamline the tracking plan by allowing variations of an event to suit different scenarios, making the specification more precise. Each variant, inheriting properties from a base event, can be modified and enhanced. This approach simplifies both implementation and communication with developers, as it narrows the scope of changes to only necessary areas. Additionally, event variants generate their own Codegen functions, facilitating quick and accurate implementation by developers and reducing uncertainty about property values. Existing events with numerous conditional properties used across various scenarios can greatly benefit from event variants.
+Event variants streamline the tracking plan by allowing variations of an event to suit different scenarios, making the specification more precise. Each variant, inheriting properties from a base event, can be modified and enhanced.
+
+### Developers love variants
```
[the remark this answers is not in the document]
Comment (hugihlynsson) [reply pages/data-design/avo-tracking-plan/event-variants.mdx:21]: I like it!

**Review by hugihlynsson — APPROVED** [review]
Nice!

**Pull request merged**